Cargo.toml

```diff
@@ -33,27 +33,32 @@ authors = ["yavko <yavornkolev@gmail.com>"]
 
 [dependencies]
 hyprland-macros = { path = "hyprland-macros", version = "0.3.4" }
-serde = {version = "1", features = ["derive"]}
+serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 serde_repr = "0.1"
-hex = "0.4"
-tokio = { version = "1", features = ["full"], optional = true }
-async-net = { version = "2", optional = true }
-async-std = { version = "1", optional = true }
-futures-lite = { version = "2", optional = true }
-lazy_static = "1.4"
-regex = "1.10"
-async-trait = "0.1"
-num-traits = "0.2"
-doc-comment = "0.3"
-paste = "1"
-strum = { version = "0.26", features = ["derive"] }
-derive_more = "0.99"
-futures = "0.3"
+hex = "0.4.3"
```

Comment:
U can remove it cuz u removed the only method that used it

Reply:
I removed it, but the method `Address::as_vec` requires it, so I commented that out. (it wasn't used in the library internally) If we are to add it back, there seems to be a better crate for that -- https://crates.io/crates/const-hex